src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/IEnvironmentOptions.cs

```diff
@@ -27,5 +27,22 @@ internal interface IEnvironmentOptions
         ///     The value to return if the value does not exist.
         /// </param>
         T GetOption<T>(string category, string page, string option, T defaultValue);
+
+        /// <summary>
+        ///     Sets a value associated with the specified category, page and option, if it exists,
+        /// </summary>
+        /// <param name="category">
+        ///     A <see cref="string"/> containing the category of the option to return.
```

Comment:
I think you mean "the option to set" rather than "the option to return". Also affects the next two parameters.